lib/result.js

```diff
@@ -97,18 +91,11 @@ Result.prototype.addFields = function (fieldDescriptions) {
     this.fields = []
     this._parsers = []
   }
-  var ctorBody = ''
   for (var i = 0; i < fieldDescriptions.length; i++) {
     var desc = fieldDescriptions[i]
     this.fields.push(desc)
     var parser = this._getTypeParser(desc.dataTypeID, desc.format || 'text')
     this._parsers.push(parser)
-    // this is some craziness to compile the row result parsing
-    // results in ~60% speedup on large query result sets
-    ctorBody += inlineParser(desc.name, i)
```

Comment:
if you guys are dead set against this pull, then at least one improvement that could be definitely made to pg is to make ctorBody an array, have this line push the inline parser string into the array and then do an array.join at the end, currently here we are making 2 strings for each field of the results